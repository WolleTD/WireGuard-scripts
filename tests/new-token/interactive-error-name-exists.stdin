peer1
peer2
172.16.1.2/32
n
