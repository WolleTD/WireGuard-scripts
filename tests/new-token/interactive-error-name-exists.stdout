Add wg-setup tokens to /tmp/tmp\..+/toklist
Name: A peer with this Name already exists!
Name: AllowedIPs: 
[A-Z2-7]{8}		172\.16\.1\.2/32		peer2
Add another token\? \[y/N\] 
